### PR TITLE
release: Epic/boba

### PR DESCRIPTION
**Pull request by Tulun**
Merge adds BOBA to list of potential L2->L1 options.

Merge when want to release to production.

### CODE REVIEW

**Review by daywiss — APPROVED** [review]
lgtm

File: src/state/send.ts [hunk 1]
```diff
@@ -85,9 +85,13 @@ const sendSlice = createSlice({
...
       })
       .addCase(toggleConfirmationScreen, (state, action) => {
-        // If the confirmation screen is closed, reset the state.
+        // If the confirmation screen is closed, reset some values in the state.
         if (action.payload.showConfirmationScreen === false) {
-          state = initialState;
+          state = {
+            ...state,
+            amount: ethers.constants.Zero,
+            error: undefined,
```
Comment (mrice32) [inline src/state/send.ts:93]: These and other changes are minor fixes I presume and are unrelated to Boba?

**Review by mrice32 — APPROVED** [review]
LGTM!

File: src/Routes.tsx [hunk 2]
```diff
@@ -87,3 +89,9 @@ const Routes: FC<Props> = () => {
 };
 
 export default Routes;
+
+const RemoveErrorSpan = styled.span`
+  cursor: pointer;
+  font-size: 1rem;
```
Comment (Gamaranto) [inline src/Routes.tsx:95]: nit, can remove this

File: src/Routes.tsx [hunk 3]
```diff
@@ -87,3 +89,9 @@ const Routes: FC<Props> = () => {
...
+  font-weight: 600;
```
Comment (Gamaranto) [inline src/Routes.tsx:96]: nit, we only import weights for 400 and 700, so I'm not sure what this declaration would do, it might do one of the three depending on browser:
- Interpolate to a synthetic 600 weight instead of using the bold variant of the font
- Take the closest variant with a font-weight set (in this case the bold version of the font set with weight 700) and use that
- do nothing and keep the standard (400)

TL;DR we don't import 600 so this declaration behaves in an undefined way, lets set it to 700 to get a true bold

**Review by Gamaranto — COMMENTED** [review]
2 some minor nits, otherwise lgmt

File: src/state/send.ts (hunk 1 above)
Comment (daywiss) [reply src/state/send.ts:93]: yea this was related to the form state changing after returning from the confirmation page. Probably should have gone into master. https://app.shortcut.com/uma-project/story/3606/update-the-default-chain-for-sending-assets-from-on-across
